crates/rolldown/tests/rolldown/misc/footer/cjs/_config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this test case related?

Reply:
Nope, it seems that I brought it from the previous branch before this :) Is it necessary to remove it?

Reply:
Let's remove it first.

Reply:
Okay, I'll remove it later (hanging out now).